app/models/collection.rb

```diff
@@ -4,4 +4,5 @@
 # Models a collection in the database
 class Collection < ApplicationRecord
   has_many :works, dependent: :destroy
+  belongs_to :creator, class_name: 'User'
```

Comment:
Will Rails automatically validate this?

Reply:
yes. because we don't have `optional: true`

Reply:
Perfect. Resolving.